synapseclient/models/curation.py

```diff
@@ -1148,7 +1148,7 @@ class GridSynchronousProtocol(Protocol):
 
     def create(
         self,
-        attach_to_previous_session=True,
+        attach_to_previous_session=False,
```

Comment:
I made this change because if you do have a previous session open, and you are not aware that this defaults to `True` it could drop you into a session you did not intend to and overwrite, or delete data in the RecordSet.

Reply:
Hmm. I actually think the backend is going to change where the default is one session, and multiple people can connect and iterate on that one session.

We can figure it out then